doc/amazon_sagemaker_processing.rst

```diff
@@ -27,37 +27,38 @@ You can run notebooks on Amazon SageMaker that demonstrate end-to-end examples o
 Data Pre-Processing and Model Evaluation with Scikit-Learn
 ==================================================================
 
-You can run a Scikit-Learn script to do data processing on SageMaker using the `SKLearnProcessor`_ class.
-
-.. _SKLearnProcessor: https://sagemaker.readthedocs.io/en/stable/sagemaker.sklearn.html#sagemaker.sklearn.processing.SKLearnProcessor
+You can run a Scikit-Learn script to do data processing on SageMaker using the :class:`sagemaker.sklearn.processing.SKLearnProcessor` class.
```

Comment:
`@ignorable` small doc nit: you typically see in scikit-learn.org [docs](https://scikit-learn.org/stable/getting_started.html), the L is never uppercase. The S is typically only uppercase at the beginning of sentences.